src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -1073,6 +1073,20 @@ protected virtual float AdjustTargetPositionOfScrollAnimation(float position)
             return position;
         }
 
+        /// <summary>
+        /// Current scroll position given to ScrollTo.
```

Comment:
CurrentPosition is not exactly same as "position given to ScrollTo".
It is just current position so when user use SrollTo Api to 4000 and get CurrentPosition at the middle of animation, the value can be "current position", not "position given to ScrollTo".

If you really want to return "position given to ScrollTo", I think Position property is much suitable because it changed to destination value when starting ScrollTo Animation.

Reply:
I understand your point.

How about supporting ScrollPosition and ScrollCurrentPosition? (same as ContentContainer.Position and ContentContainer.CurrentPosition)
- ScrollableBase.ScrollPosition = -ScrollableBase.ContentContainer.Position
- ScrollableBase.ScrollCurrentPosition = -ScrollableBase.ContentContainer.CurrentPosition

What do you think about the above?

Reply:
I think user always wants to get CurrentPosition instead of Position.

So, just update comment will be fine.
ScrollPosition = -ScrollableBase.ContentContainer.CurrentPosition.

What do you think?

Reply:
I am not sure if it is sufficient.
Because View class already supports "CurrentPosition" and "Position" to support position information separately. (i.e. visible position and property position)
So I think it would be better that ScrollPosition should be supported in the same way. Otherwise, user cannot get the property position for scrolling.

Reply:
As you mentioned before, please add two APIs

- ScrollableBase.ScrollPosition = -ScrollableBase.ContentContainer.Position
- ScrollableBase.ScrollCurrentPosition = -ScrollableBase.ContentContainer.CurrentPosition

Reply:
I have updated the patch.
However, for ScrollPosition, I could not assign -ContentContainer.Position because it was not the same with the parameter given to ScrollTo(). (It seems there is a bug to be fixed about ContentContainer.Position)
So I used private variable scrollToPosition to track the parameter of ScrollTo.